backend-tests/tests/test_useradm_enterprise.py

```diff
@@ -124,10 +122,25 @@ def test_enable_disable(self, tenants_users):
         totp = pyotp.TOTP(secret)
         tok = totp.now()
 
+        # verify token
+        r = self._verify(user_2fa_tok, tok)
+        assert r.status_code == 202
+
         # login with totp succeeds
         r = self._login(user_2fa, totp=tok)
         assert r.status_code == 200
 
+        r = self._login(user_2fa)
```

Comment:
I would add here the comment partially removed above: "token has been verified, login shall now work anymore" (or something like that)

Reply:
added.

Reply:
s/logi/login :)